.travis.yml

```diff
@@ -113,3 +136,8 @@ install: true  # Skip the installation step, as Maven requires
 script:
   - cd "$TRAVIS_BUILD_DIR"
   - .travis/run_travis_job.sh
+
+notifications:
+  email:
```

Comment:
Email notifications are not triggered for Pull Requests builds, so we will (hopefully) receive notifications for cron builds and master builds only.